node/network/statement-distribution/src/requester.rs

```diff
@@ -161,6 +165,7 @@ pub async fn fetch(
 					);
 
 					metrics.on_received_response(false);
+					metrics.on_unexpected_statement(UNEXPECTED_STATEMENT_LARGE_LABEL);
```

Comment:
nit: we probably want to move the const to the `metrics.rs` side, and only expose `on_unexpected_statement_large()` and friends, this reduces visual complexity quite a bit. I promise this is the last thing :) - if that's resolved, feel free to merge as is :+1: